mitmproxy/tcp.py

```diff
@@ -47,5 +50,10 @@ def __init__(self, client_conn, server_conn, live=None):
         messages=List[TCPMessage]
     )
 
+    def copy(self):
```

Comment:
No, I would get back the same message object instance - which I don't want. I want a deep-copy.